packages/react-liveness/src/components/FaceLivenessDetector/service/utils/blazefaceFaceDetection.ts

```diff
@@ -23,13 +25,15 @@ export class BlazeFaceFaceDetection extends FaceDetection {
 
   constructor(binaryPath?: string, faceModelUrl?: string) {
     super();
-    this.faceModelUrl = faceModelUrl;
+    this.faceModelUrl =
+      faceModelUrl ??
+      `https://cdn.liveness.rekognition.amazonaws.com/face-detection/tensorflow-models/blazeface/${BLAZEFACE_VERSION}/model/model.json`;
```

Comment:
Can we put both of these on constants with giant comments saying where to update docs and that it's a breaking change to update it?